packages/ckeditor5-dev-release-tools/lib/tasks/releasesubrepositories.js

```diff
@@ -873,7 +873,8 @@ module.exports = async function releaseSubRepositories( options ) {
 			repositoryOwner: releaseDetails.repositoryOwner,
 			repositoryName: releaseDetails.repositoryName,
 			version: `v${ releaseDetails.version }`,
-			description: releaseDetails.changes
+			description: releaseDetails.changes,
+			isPrerelease: npmTag !== 'latest'
```

Comment:
What if we publish only on GH? We can pass the modifier, but publishing a release on GH only and passing an npm version does not make sense.

I would change the implementation to assert the returned value from `semver.prerelease( version )`.

Does it make sense?

Reply:
Of course, thanks for noticing such a scenario.